docs/source/branch.rst

```diff
@@ -0,0 +1,218 @@
+model parameters. Without any extra tooling this would require manually manipulating the configuration file and
+re-running for each desired change which would quickly get out of hand. The branches file helps us do this in a
+convenient way. This section will detail the common ways simulations are varied and the different aspects of a branches
+file that help us do this.
+
+Uncertainty
+-----------
+
+Generating uncertainty for results is a core tenant of IHME and this is no different for simulation science. We are
+primarily concerned with uncertainty from two sources in our models -- uncertainty surrounding the data that goes in to
+a model and uncertainty due to stochasticity. The branches file can help us explore these uncertainties easily by
+allowing us to vary the input draws from the Global Burden of Disease (GBD) used as well as the random seeds used by the
+simulation.
+
+Data Uncertainty
+^^^^^^^^^^^^^^^^
+Our simulations primarily rely on GBD results which are produced in draws to estimate uncertainty. In order to reflect
+the GBD's uncertainty surrounding their results in our simulations, we generally run simulations with a variety of
+output draws.
+
+.. note::
+    A draw is a statistical term related to bootstrapping that has a specific meaning in the context of the GBD. The
+    implementation details vary, but the purpose is for
+
+    some quantity or measure of interest, a draw is a member of a set a full set of results such that, when taken
+    together, the set of draws describes at least some of the uncertainty surrounding the quantity as a result of the
+    modeling process, data uncertainty, etc. Generally, GBD results are produced in sets of 1000 draws.
+
+To do this, we can use the ``input_draw_count`` key in a branches file. This key specifies an integer that represents
+the number of different simulations that will be run in parallel, each with a different draw of GBD results drawn
+uniformly from the number available, which is generally 1,000. Below is a very simple branches file containing only a
+mapping that dictates the number of draws to run.
+
+.. code-block:: yaml
+
+    input_draw_count: 10
+
+Stochastic Uncertainty
+^^^^^^^^^^^^^^^^^^^^^^
+Vivarium simulations are probabilistic in nature. They tend to contain distributions that describe quantities rather
+than singular values. Because of this, our models are subject to stochastic uncertainty. To attempt to capture some of
+this uncertainty we can run the simulation with different random seeds which will result in different numbers drawn from
+the underlying pseudo-random number generator, and thus different output results. This can be specified in the branches
+file with the ``random_seed_count`` key. This key specifies an integer that represents the number of different random
+seeds to use, each generated randomly and run in a separate simulation.
+
+.. note::
```

Comment:
I'm just looking at the 'View file' option on github which doesn't render this notes nicely - do they work better with sphinx?

Reply:
not until I fix the deps

Reply:
ok, see below